src/App.tsx

```diff
@@ -1,9 +1,25 @@
-import { HelloHell } from "./components/HelloHell";
+import { FrontendMentorMobileText, NoFeedbackMobileText } from "components/common/typography";
+import { HelloHell } from "components/HelloHell";
+import { Shelf, Spacer } from "components/common/layout";
```

Comment:
```suggestion
import { Shelf, Spacer } from "components/common/layout";
import { FrontendMentorMobileText, NoFeedbackMobileText } from "components/common/typography";

import { HelloHell } from "components/HelloHell";
```

Layout should some before typography and any other component. Maybe the basic building components should have a visual division from the components.

Reply:
Makes sense